src/openrct2/ride/Vehicle.cpp

```diff
@@ -1945,6 +1945,21 @@ static SoundIdVolume sub_6D7AC0(
     return { currentSoundId, currentVolume };
 }
 
+SoundIdVolume Vehicle::GetLiftHillSound(Ride* curRide)
```

Comment:
Do you have a preference for the name of this function, since there seems to be an effort to not name it sub_6xxxxx?